speakers.js

```diff
@@ -0,0 +1,67 @@
+const speakers = [
+  {
+    name: 'james',
+    subtittle: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+    src: './images/hll.jpg',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. ',
+  },
+  {
+    name: 'chriss',
+    subtittle: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+    src: './images/yoo.jpg',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. ',
+  },
+  {
+    name: 'jadden',
+    subtittle: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+    src: './images/thant.jpg',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. ',
+  },
+  {
+    name: 'jadden',
+    subtittle: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+    src: './images/yoo.jpg',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. ',
+  },
+  {
+    name: 'jadden',
+    subtittle: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+    src: './images/hll.jpg',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. ',
+  },
+  {
+    name: 'jadden',
+    subtittle: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+    src: './images/thant.jpg',
```

Comment:
- [ ] Your page should look almost identical to the original design, outside the content differences. It should have exact color, background, borders, and font styles. 
Fonts that were originally used : Cocogoose, Lato


To your specific example it would look something like this:

Your design            |  Behance design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/43877689/175785380-703ce88f-0c9a-435c-a471-930bf0537440.png)  |  ![image](https://user-images.githubusercontent.com/43877689/175785398-17e44820-b53e-48f6-b039-7377bf214d84.png)